src/data/user/user.state.ts

```diff
@@ -11,4 +11,5 @@ export interface UserState {
   location?: GeoPoint;
   client?: Client;
   isClientConnected: boolean;
+  visibility?: string;
```

Comment:
this should actually be in session, but that would be an annoying fix. Up to you if you want to move it.